Dockerfile

```diff
@@ -53,6 +53,27 @@ EXPOSE 8000
 ENTRYPOINT ["./docker-entrypoint.sh"]
 CMD ["python", "./cfgov/manage.py", "runserver", "0.0.0.0:8000"]
 
+# Build Frontend Assets
+FROM cfgov-dev as cfgov-build
+
+ENV STATIC_PATH ${APP_HOME}/cfgov/static/
+ENV PYTHONPATH ${APP_HOME}/cfgov
+
+# Django Settings
+ENV DJANGO_SETTINGS_MODULE cfgov.settings.production
+ENV DJANGO_STATIC_ROOT ${STATIC_PATH}
+ENV ALLOWED_HOSTS '["*"]'
```

Comment:
Is there any way to avoid the duplication of these variables between the `cfgov-dev`, `cfgov-build`, and `cfgov-prod`?

Reply:
I can play with it - I think if I define them all in `cfgov-dev` they should propogate.

Reply:
although - I think some of them need to be duplicated, like the django settings module as we don't want to overrite the dev value.

Reply:
Yeah. We just duplicate a few of them in the Dockerfile. But if there's not an easy/obvious way to reuse them, I don't want that question to block this.

Reply:
@willbarton upon first attempt looks like it doesn't like passing them down through the parent - sepcifically it wasn't able to find the `STATIC_ROUTE` shared variable when it is defined in the initial cfgov-dev container. For now I think limited repeatability is ok